identity-and-access-management/secret-manager/how-to/create-folder.mdx

```diff
@@ -22,20 +22,45 @@ This page explains how to create [folders](/identity-and-access-management/secre
   - You have [generated an API key](/identity-and-access-management/iam/how-to/create-api-keys/) and enabled the `SecretManagerFullAccess` [permission set](/identity-and-access-management/iam/reference-content/permission-sets/)
 </Message>
 
-## How to create a folder
 
-1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
-2. Click **Create folder**.
-3. Enter a name for your folder and click **Create folder**. Your newly created folder displays.
+<Tabs id="install">
+  <TabsTab label="Create your first folder">
 
-## How to create a subfolder
+   <Message type="note">
+     If you have no resources in Secret Manager yet, you can create a folder when creating your secret.
+   </Message>
 
-1. [Create a folder](#how-to-create-a-folder).
-2. Click the folder in which you want to create a subfolder.
-3. Click **Create folder**.
-4. Enter a name for your subfolder.
-5. Click **Create folder**. Your subfolder displays inside your folder.
+    1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+    2. Click **+ Create secret**.
+    3. Choose the [region](/identity-and-access-management/secret-manager/concepts/#region) in which you want to store your folder.
+        <Message type="important">
+         Folders cannot be moved from one region to another after creation.
+        </Message>
+    4. Select **Create a new path** and enter `/` followed by a name for your [folder](/identity-and-access-management/secret-manager/concepts/#folder). For example `/my-folder` if you want to create a folder named `my-folder`.
+        <Message type="important">
+          A [path](/identity-and-access-management/secret-manager/concepts/#path) is the directory structure to access your folders and secrets. Each path **must be prefixed** with a slash.
+        </Message>
+    5. Enter a name for your secret and add tags (optional).
+    6. Add your secret manually or import it from a file.
+        <Message type="note">
+         The maximum file size for your secret is 64 KiB.
+        </Message>
+    7. Click **Create secret**. The **Overview** tab of your secret displays.
+    8. Click **Back to Secrets** to display your folder and its contents.
+  </TabsTab>
+  <TabsTab label="Create more folders">
+    1. Click **Secret Manager** in the **Security and Identity** section of the [Scaleway console](https://console.scaleway.com/) side menu.
+    2. Select your desired [region](/identity-and-access-management/secret-manager/concepts/#region) in the **Region** drop-down.
+         <Message type="important">
+         Folders cannot be moved from one region to another after creation.
```

Comment:
Yes, this is because they will only display when you click the relevant tab. So in reality, the message only displays once per tab. 
<img width="816" alt="Capture d’écran 2023-11-23 à 10 55 49" src="https://github.com/scaleway/docs-content/assets/87707325/af5f2896-cb99-4f0c-b553-9a4aaa82c0d2">
<img width="816" alt="Capture d’écran 2023-11-23 à 10 55 41" src="https://github.com/scaleway/docs-content/assets/87707325/45561442-f5d4-41c2-8168-bd09151ca777">

Reply:
Ok perfect then 👍